src/utils/index.ts

```diff
@@ -0,0 +1,50 @@
+import * as jwt from 'jsonwebtoken';
+import nodemailer = require('nodemailer');
+import { jwtSecret, mail, url } from '../config/config';
+import { codeErrors } from '../config/config';
+import { SecureError } from './errors';
+
+const transport = nodemailer.createTransport(mail);
+
+export const sendMail = (email: string, name: string): void => {
+  const token: any = getToken({ email });
+  const html = generateEmail(name, email, token);
+  transport.sendMail(
+    {
+      from: 'Car Fdinder',
+      html,
+      subject: 'Confirma registration',
+      to: email
+    },
+    err => {
+      if (err) {
```

Comment:
Please add logging here. Also this throw won't be handled anywhere and will crash the stack if I'm not mistaken.